sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 ## Release History
 
+### 4.49.0-beta.1 (Unreleased)
```

Comment:
will also need to update the azure-cosmos sdk version in pom.xml to 4.49.0-beta.1, else the pipeline will fail

and then you will need to also update all the versions mentioned in 
`{x-version-update;com.azure:azure-cosmos;dependency} `
`{x-version-update;com.azure:azure-cosmos;current}`

Reply:
you should eb able to use azure-sdk-for-java\eng\versioning update_versions.py if you don't want to do it manually

Reply:
Ran the prepare release script for this to update required files to use 4.48.3-hotfix.